### ISSUE TITLE
v10 - Accessibility issues with headings

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#732]
## Environment
Usability testing with JAWS on Windows 10

## Problem
- Headings should be in strict hierarchical order on the page 
- If possible each page should have only one H1
- Possibly too many headings used
_User cited "Inclusive, flexible, and consistent" on the homepage as being not really being a H3_

**Comment by alisonjoseph** [comment]
fyi @jeanservaas

**Comment by jeanservaas** [comment]
Hey @alisonjoseph I can't see the code structure of the page in markdown in VS Code but this seems like a problem we could fix by referring to these headings differently in code — i.e. we don't want to override the style of the page, we just want to make sure that the screen reader reads the headings in the correct order.

**Comment by alisonjoseph** [comment]
`# this is h1`
`## this is h2`
`### this is h3`

and so on. we are good in that we only use h1s at the top of the page, but we can probably do better elsewhere through the site.

**Comment by jeanservaas** [comment]
right, yeah -- the way i understood this issue was that on the homepage, the screen reader was reading the larger type first even though it was not the thing that was meant to be read first. 

I guess I thought we were talking about the homepage because of this comment, but maybe I misunderstood... 

_User cited "Inclusive, flexible, and consistent" on the homepage as being not really being a H3_

Are we talking about the entire site? I know there are hierarchy issues on the code tab of the components (Read me) stuff too, because we don't have an H5...

**Comment by dakahn** [comment]
Testing with February 2019 ruleset and this is no longer an issue 👍🏽
